docs/installation.rst

```diff
@@ -50,12 +50,19 @@ Execute the following command to install latest DALI for specified CUDA version:
 
    pip install --extra-index-url https://developer.download.nvidia.com/compute/redist nvidia-dali-cuda100
 
-* for CUDA 11:
+* for CUDA 11 and CUDA 11.1:
+
+  .. note::
+    CUDA 11.1 build uses CUDA toolkit enhanced compatibility. It is built with the latest CUDA 11.x
+    toolkit while it can run on CUDA 11.0 capable drivers 450.80 or later. Using the newest driver may enable
+    additional functionalities.
 
 .. code-block:: bash
 
    pip install --extra-index-url https://developer.download.nvidia.com/compute/redist nvidia-dali-cuda110
 
+   pip install --extra-index-url https://developer.download.nvidia.com/compute/redist nvidia-dali-cuda111
```

Comment:
I'm thinking about splitting this to separate 11.0 and 11.1

Reply:
Done